grunt-configs/eslint.js

```diff
@@ -4,71 +4,19 @@ module.exports = function(grunt, options) {
         self: [
             'Gruntfile.js'
         ],
-        es6: {
+        all: {
             files: [{
                 expand: true,
-                cwd: 'static/src/javascripts/es6',
+                cwd: 'static/src',
                 src: [
                     '**/*.js'
                 ]
-            }]
-        },
-        test: {
-            files: [{
-                expand: true,
-                cwd: 'static/src/javascripts/test',
-                src: [
-                    '**/*.js'
-                ]
-            }]
-        },
-        common: {
-            files: [{
-                expand: true,
-                cwd: 'static/src/javascripts/projects/common',
-                src: [
-                    '**/*.js',
-                    '!utils/atob.js'
-                ]
-            }]
-        },
-        facia: {
-            files: [{
-                expand: true,
-                cwd: 'static/src/javascripts/projects/facia',
-                src: [
-                    '**/*.js'
-                ]
-            }]
-        },
-        'facia-tool': {
```

Comment:
Sory, do you mean this?

```
        all: {
            files: [{
                expand: true,
                cwd: 'static/src',
                src: [
                    '**/*.js'
                ]
            }],
```

That doesn't match facia-tool `cwd: 'facia-tool/public/js/',`